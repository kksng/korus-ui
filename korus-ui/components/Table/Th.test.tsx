import React from 'react';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';
import { Th } from './Th';

describe('Th component tests:', () => {
  it('should be defined', () => {
    expect(Th).toBeDefined();
  });
  it('should render', () => {
    const wrapper = shallow(<Th>test</Th>);

    expect(wrapper.text()).toEqual('test');
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
