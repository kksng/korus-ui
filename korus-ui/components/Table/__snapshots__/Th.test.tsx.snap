// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Th component tests: should render 1`] = `
<th>
  test
</th>
`;
